### DISCUSSION TITLE
How to set up logger in processor

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1273] [Dev] [answered]
Related to #1262

I realized that I did not get any log output to my debug console anymore

I initialized the logger via

`private static final Logger LOG = LoggerFactory.getLogger(LatLngToJtsPointProcessor.class);`

With following code, I did not get any output at all
`LOG.debug("Created Geometry: " + geom.toString());` 

This is an "older code" so pretty sure that this was working in the past.

Is there a new approach with the 1 class processor?
Or do I miss additional setting requirements?

I looked in other processors but see no major differences because I am pretty certain that there is no output as well.

Example code here:
https://github.com/apache/streampipes/blob/238c7cd082f132f03498ab4ff4ff8d50132d78f8/streampipes-extensions/streampipes-processors-geo-jvm/src/main/java/org/apache/streampipes/processors/geo/jvm/jts/processor/latlngtojtspoint/LatLngToJtsPointProcessor.java#L105

**Comment by tenthe** [comment]
Hi @flomickl,
your logging code looks good. I suspect this is related to the changes in this PR #1267.

**Comment by dominikriemer** [comment, marked as answer]
Yes, our default log level is `info`. You can change the level by providing the environment variable (I think it's `-Dlogback.debug=true`) or change the logback configuration file.

**Comment by flomickl** [comment]
Ah!  Good to know.
Can you describe how to set it up in IntelliJ?
I tried it as environment variable, but it is not working.

**Reply by tenthe** [reply]
Hi, you can set the log level with the following environment variable in IntelliJ:
`logging.level.root=WARN`.
Maybe we can also include this information in the description of the developer setup in the wiki.

**Comment by flomickl** [comment]
ah cool.
It is working as expected
I added it here https://cwiki.apache.org/confluence/x/9YoODg

**Reply by flomickl** [reply]
I started a new discussion about this general topic here  #1325
